core/app_server.py

```diff
@@ -249,17 +232,17 @@ def handle(self, exception):
                 # service. It's a serious problem, but probably not
                 # indicative of a bug in our software. Log it at log level
                 # WARN.
-                log_method = logging.warning
+                log_method = self.log.warning
             response = make_response(document.response)
         else:
             # There's no way to turn this exception into a problem
             # document. This is probably indicative of a bug in our
             # software.
-            if debug:
+            if self.debug:
                 body = tb
             else:
-                body = _("An internal error occured")
-            response = make_response(str(body), 500, {"Content-Type": "text/plain"})
+                body = "An internal error occured"
```

Comment:
Good catch! Since I just changed this line, and didn't actually change the text, out of curiosity I went back to see when this typo actually entered the code base, looks like it came in on July 15, 2015 https://github.com/ThePalaceProject/circulation/commit/57bae60d90c70bd6fd9740d92406785bf15e9d83. Good one to fix 😄